pexpect/pty_spawn.py

```diff
@@ -752,10 +752,14 @@ def interact(self, escape_character=chr(29),
         child process in interact mode is duplicated to the given log.
 
         You may pass in optional input and output filter functions. These
-        functions should take a string and return a string. The output_filter
-        will be passed all the output from the child process. The input_filter
-        will be passed all the keyboard input from the user. The input_filter
-        is run BEFORE the check for the escape_character.
+        functions should take bytes array and return bytes array too. Even
+        after Python 3 ``unicode='utf-8'`` support, meth:`interact` doesn't
```

Comment:
Do you mean `encoding='utf-8'`?

Also, this is not specific to Python 3; the same is true in Python 2 (with `unicode` in place of `str`).

Reply:
Will fix